src/styles/Colors.ts

```diff
@@ -29,12 +29,12 @@ enum Colors {
   pink500 = '#EC4899',
   pink300 = '#F9A8D4',
 
-  lime900 = '#365314',
-  lime700 = '#4D7C0F',
-  lime600 = '#65A30D',
-  lime500 = '#84CC16',
-  lime400 = '#A3E635',
-  lime300 = '#BEF264',
+  lime900 = '#14532D',
```

Comment:
Shouldn't these colors be changed in figma as well?

Reply:
of course :) 
this was just a trial to see the potential implications of a change (via vercel previews) :) 

will revert them